data/gui/window/editor_edit_pbl.cfg

```diff
@@ -0,0 +1,373 @@
+#textdomain wesnoth-lib
+###
+### Definition of the window to edit an add-ons _server.pbl
+###
+
+#define ROW INPUT LABEL ID TOOLTIP
+[row]
+
+	[column]
+		border = "all"
+		border_size = 5
+		horizontal_alignment = "left"
+
+		[label]
+			id={ID}+"_label"
+			definition = "default"
+			label = {LABEL}
+		[/label]
+	[/column]
+
+	[column]
+		border = "all"
+		border_size = 5
+		horizontal_grow = true
+
+		[{INPUT}]
+			id = {ID}
+			definition = "default"
+			tooltip = {TOOLTIP}
+		[/{INPUT}]
+	[/column]
+
+[/row]
+#enddef
+
+[window]
+	id = "editor_edit_pbl"
+	description = "PBL editor."
+
+	[resolution]
+		definition = "default"
+
+		automatic_placement = true
+		vertical_placement = "center"
+		horizontal_placement = "center"
+
+		maximum_width = 800
+
+		[linked_group]
+			id = "language"
+			fixed_width = true
+		[/linked_group]
+
+		[linked_group]
+			id = "title"
+			fixed_width = true
+		[/linked_group]
+
+		[linked_group]
+			id = "description"
+			fixed_width = true
+		[/linked_group]
+
+		[tooltip]
+			id = "tooltip"
+		[/tooltip]
+
+		[helptip]
+			id = "tooltip"
+		[/helptip]
+
+		[grid]
+
+			[row]
+
+				[column]
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+
+					[label]
+						id = "title"
+						definition = "title"
+						label = _ "PBL Setup"
+					[/label]
+				[/column]
+
+			[/row]
+
+			[row]
+
+				[column]
+					horizontal_grow = true
+
+					[grid]
+
+						{ROW text_box _"Name:" "name" _"The name of the add-on displayed in-game."}
+						{ROW text_box _"Description:" "description" _"The description of the add-on to display in the add-ons manager."}
+						{ROW text_box _"Icon:" "icon" _"The path to the icon image to display to display in the add-ons manager. Must be an image from mainline."}
+						{ROW text_box _"Author:" "author" _"The author of this add-on. If forum authentication is used, this is your forum username."}
+						{ROW text_box _"Version:" "version" _"The add-on's current version. This should be of the form X.Y.Z."}
```

Comment:
Typographical apostrophe.

Version numbers can also be just X.Y, see #7396.

Reply:
> Version numbers can also be just X.Y,

And also support an arbitrary suffix, so it's more accurate to say they should _start_ with X.Y or X.Y.Z.

Reply:
Fixed the apostrophes.

The word "This should be of the form X.Y.Z." is meant to be simple more than it's meant to describe how wesnoth's versioning works.